imports/questionnaire_imports.json

```diff
@@ -1580,847 +1209,505 @@
         "description_de": "",
         "description_fr": "",
         "description_it": ""
       },
       {
         "questions": [],
         "id": 13,
-        "name_de": "Qualität im Leitungsteam ",
-        "name_fr": "Qualité de la maîtrise  ",
-        "name_it": "Qualità nel gruppo animatori",
-        "description_de": "Verschiedene Aspekte wie die Aus- und Weiterbildung oder die Motivation und Stimmung beeinflussen die Qualität im Leitungsteam. Eine gute Qualität im Leitungsteam wirkt sich positiv auf die Qualität des Pfadiprogramms aus.  ",
-        "description_fr": "Divers aspects tels que la formation initiale et continue ou la motivation et la bonne humeur influencent la qualité de la maîtrise. Une maîtrise de qualité a un effet positif sur la qualité du programme scout.",
-        "description_it": "Vari aspetti come la formazione di base e la formazione continua o la motivazione e l'umore influenzano la qualità del gruppo animatori. Una buona qualità nella direzione ha un effetto positivo sulla qualità del programma scout."
+        "name_de": "Ausgebildete Leiter*innen",
+        "name_fr": "Responsables formé·e·s ",
+        "name_it": "Capi formati ",
+        "description_de": "",
+        "description_fr": "",
+        "description_it": ""
       },
       {
         "questions": [],
         "id": 14,
-        "name_de": "Motivation Leitende ",
-        "name_fr": "Motivation des responsables  ",
-        "name_it": "Motivazione degli animatori",
+        "name_de": "Vernetzung Abteilungen",
+        "name_fr": "Mise en contact des groupes ",
+        "name_it": "Contatti tra sezioni",
         "description_de": "",
         "description_fr": "",
         "description_it": ""
-      },
-      {
-        "questions": [],
-        "id": 15,
-        "name_de": "Ressourcen ",
-        "name_fr": "Ressources  ",
-        "name_it": "Risorse",
-        "description_de": "Um ihre Aufgaben erfüllen zu können, ist eine Abteilung auf Ressourcen angewiesen, dies umfasst materielle, personelle sowie finanzielle Mittel. Je mehr Mittel eine Abteilung hat, desto umfangreichere Aktivitäten können angeboten werden. In der Pfadi hängen diese Mittel häufig direkt mit der Mitgliederzahl zusammen.  ",
-        "description_fr": "Afin de pouvoir remplir ses fonctions, un groupe dépend de ressources, qu'il s'agisse de ressources matérielles, humaines ou financières. Plus un groupe dispose de moyens, plus les activités proposées sont variées. Chez les scouts, ces ressources sont souvent directement liées au nombre de membres.",
-        "description_it": "Per essere in grado di svolgere i suoi compiti, una sezione dipende dalle risorse, che includono risorse materiali, umane e finanziarie. Più risorse ha una sezione, più variegate possono essere le attività offerte. Nello scoutismo, questi mezzi sono spesso direttamente correlati al numero di attivi."
       }
-    ]
+    ],
+    "type": "Questionnaire::Group::Default"

```

Comment:
@SebastianStorz please move the type to be the first element of this object. This makes it easier to understand which questionnaire your looking at when opening this file